VTUL/VTDLP-sitecontent
# Renamed html files with standard names

Jira ticket: https://webapps.es.vt.edu/jira/browse/LIBTD-2288

What does this PR do?
Renames html files with standard names.

How to test?
Once I submit a PR for the VTDLP-siteconfig repo to update the html file names there, the html should continue to pull from these files as usual.

Interested parties:
@yinlinchen
